src/components/PageComponents.js

```diff
@@ -60,15 +63,34 @@ class PageComponents extends Component {
     this.props.dispatch(uiBrowseUpdateList({ update: component, value: newComponent }))
   }
 
+  facetChange(value) {
+    const activeFacets = [];
+    value && value.forEach(facet => activeFacets.push(facet['value']))
+    this.setState({ ...this.state, activeFacets})
+    console.log('Current state in Page Component:', JSON.stringify(this.state))
+  }
+
+  renderActionButton() {
+    return (<Button className='pull-right' bsStyle='success' onClick={this.filterHandler}>Filter</Button>)
```

Comment:
do we still need this Filter button? should we just trigger the filterHandle when the active facet list is changed?

Reply:
That handler does nothing at the moment anyway, so removing it.